localization/CreateStrings.lua

```diff
@@ -1,4 +1,4 @@
 for stringId, value in pairs(LLS_STRINGS) do
     ZO_CreateStringId(stringId, value)
 end
-LLS_STRINGS = nil
\ No newline at end of file
+LLS_STRINGS = ZO_ClearTable(LLS_STRINGS) -- to allow collect garbage
```

Comment:
improvement for garbagecollector

Reply:
Iterating the table should not be necessary, since it contains no tables, only strings.  The table itself is GC'd when set to nil, and there are no tables within that need GC.